buddyboss/buddyboss-platform
# Suspended users are getting email notification for group messages - Fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2727 .

### How to test the changes in this Pull Request:

1. Suspend a group member
2. Then send a group message to all group member
3. See the suspended member is not receiving email notification


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
